### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by dwickern**
See commit messages. Let me know if you want some commits as a separate PR

### CODE REVIEW

File: app/styles/ember-power-select-typeahead.scss [hunk 1]
```diff
@@ -1,8 +1,10 @@
+@import "ember-power-select";
```
Comment (cibernox) [inline app/styles/ember-power-select-typeahead.scss]: This is the only line that I think should be removed. I haven't documented it tho, but the idea is that the user imports both stylesheets:

```scss
@import "ember-power-select";
@import "ember-power-select-typeahead";
```

and in that order.

Comment (dwickern) [inline app/styles/ember-power-select-typeahead.scss]: 👍  it works even without the import. Sass variable resolution is a mystery

**Comment by cibernox** [comment]
Thanks! I've published a new version

**Comment by sly7-7** [comment]
Thanks @dwickern ❤️ 
There is just one small thing about $ember-power-select-default-border;  ember-cli complains about `Undefined variable: "$ember-power-select-default-border".`

Maybe it's just a matter of import order, but I followed what I found in https://github.com/cibernox/ember-power-select-typeahead/blob/master/tests/dummy/app/styles/app.scss

**Pull request merged**